packages/expo/src/Notifications/Notifications.ts

```diff
@@ -403,7 +403,7 @@ export default {
   },
 
   /* Primary public api */
-  addListener(listener: Function): EventSubscription {
+  addListener(listener: (notification: Notification) => unknown): EventSubscription {
```

Comment:
this might be a `LocalNotification` as well?

Reply:
Can you please verify what this should be?

Reply:
[Docs](https://docs.expo.io/versions/latest/sdk/notifications/#notificationsaddlistenerlistener) state:

>A callback that is invoked when a remote or local notification is received or selected, **with a Notification object.**

(emphasis mine)

So I think this is correct. It also matches what's currently in DefinitelyTyped. I think `LocalNotification` is only used to show a local one, the listener receives and object with the same shape (with `remote: false`, I'd assume) regardless